src/SalaryCalculator.java

```diff
@@ -1,5 +1,10 @@
 import java.util.Scanner;
 
+/**
+ * This is a small change
```

Comment:
@varkashp  - this is a bogus comment. Can you please rewrite to say what this program does?